### PR TITLE
Update LDAP Example

### PR DESCRIPTION
**Pull request by rmulley**
This PR updates the maverics LDAP example to:
* Put all services inside of Docker.
* Removes the prerequisite of golang being installed on the host machine.
* Updates the Makefile and README accordingly.

### CODE REVIEW

File: examples/ldap/docker-compose.yml [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
+      - "8899:8899"
+
   ldap:
-    image: osixia/openldap:1.2.4
+    image: osixia/openldap:1.3.0
     ports:
     - "389:389"
     - "636:636"
+
+  maverics:
```
Comment (eliasjf) [inline examples/ldap/docker-compose.yml]: Do we want to expose the Maverics Health Check Service in addition to the Authn Service in this example? At this juncture, the two are pretty independent IMO

Comment (topherMarie) [reply examples/ldap/docker-compose.yml]: I think not. One thing at a time, and we'll pull them together later

Comment (rmulley) [reply examples/ldap/docker-compose.yml]: The story asked for "an instance of the maverics app running in a docker container" - hence the inclusion.
But I can remove this if we think it's not useful or relevant enough?

Comment (rmulley) [reply examples/ldap/docker-compose.yml]: Nevermind I think I misunderstood the question 🤐

**Pull request merged**